insertionorderedmap.go

```diff
@@ -0,0 +1,25 @@
+package goflags
+
+type InsertionOrderedMap struct {
+	values map[string]*flagData
+	keys   []string `yaml:"-"`
+}
+
+func (insertionOrderedMap *InsertionOrderedMap) Get(key string) *flagData {
+	value := insertionOrderedMap.values[key]
```

Comment:
this can return nil if the value entered isn't in the map. Maybe we should handle it better here? Returning a bool along with flagData is a common pattern.

Reply:
This method is only used in within the _forEach_ which iterates through the keys themselves, because the other usages were removed, hence it cannot return nil. I will remove the method, because it's not needed anymore.

Reply:
Removing this makes sense then.